src/System.Web.Services.Description/src/System.Web.Services.Description.csproj

```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <AssemblyVersion>$(WcfAssemblyVersion)</AssemblyVersion>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
     <AssemblyName>System.Web.Services.Description</AssemblyName>
     <CLSCompliant>true</CLSCompliant>
     <IsPackable>true</IsPackable>
-    <IsBuilding>true</IsBuilding>
-    <IsHarvesting>false</IsHarvesting>
     <IsShipping>$(Ship_WcfPackages)</IsShipping>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
-	<IsPartialFacadeAssembly Condition="'$(TargetFramework)' == 'net461'">true</IsPartialFacadeAssembly>
+	  <IsPartialFacadeAssembly Condition="'$(TargetFramework)' == 'net461'">true</IsPartialFacadeAssembly>
+    <EnableDefaultItems Condition="'$(TargetFramework)' == 'net461'">false</EnableDefaultItems>
```

Comment:
Indentation is wrong on this line. First character is a tab, then there's spaces.

Reply:
Fixed via https://github.com/dotnet/wcf/pull/5223/commits/646c50673d3c5f701587f3220f564d6b85ea93a5